docs/06-concepts/18-testing/04-best-practises.md

```diff
@@ -0,0 +1,116 @@
+# Best practises
+
+## Imports
```

Comment:
suggestion: Add a description for the rule.

Does it make sense to always have a description for the rule?

Same for the new `Unit and integration tests {#unit-integration}`. Instead of having the general description in the "Don't" place it at the top.